operations/operations/zuul-ci-cd-quickstart-user-guide.md

```diff
@@ -0,0 +1,236 @@
+# Zuul users guide
+
+## Prerequisites
+
+1. Repository is known by [SCS zuul](https://zuul.scs.community)
+2. Basic ansible knowledge
+3. Basic yaml knowledge
+4. zuul-client installed
+
+Check [SCS zuul projects](https://zuul.scs.community/t/SCS/projects) for your repository to
+be available. If it is missing you need an administrator to get your repository
+configured to zuul.
+
+## Who is it for?
+
+You may have heard about zuul and may ask yourself if it is capable to support you.
+Basically everything you use ansible for can be done using zuul. That is not always
+a good thing since you may get careless and your workload will exceed the CI/CD concept.
+
+If you find yourself doing things under the following list you are at the right place.
+
+1. Code testing
+2. Deployment tests using IaC
+
+If you want to, let's say, monitor something using zuul, that is possible but not the
+intended use case.
+
+## Where do I start?
+
+Right in your projects repository! The only prerequisite you need to fulfill, is that
```

Comment:
sp: project's
no comma after fulfill (but the whole phrase "that you need to fulfill" is kind of redundant and should be dropped)

Reply:
Text dropped sp updated